### PR TITLE
Awesome books: with ES6

### PR DESCRIPTION
**Pull request by maov19**
Following features were added:

- ES6 syntax re-factoring
1. Divided your code into modules.
2. Created a separate file for each module in the modules directory.
3. Imported Books modules into the index.js file. 
4. Refactored your functions to arrow functions.
5. Kept all functionalities of the app.

- Display the current date below the navigation bar.

1. Downloaded and imported luxon library

### CODE REVIEW

File: modules/Book.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    }
+  }
+
+  removeBook(id) {
+    this.books = JSON.parse(localStorage.getItem('data'));
+    this.books = this.books.filter((i) => i?.id !== id);
+    localStorage.setItem('data', JSON.stringify(this.books));
+    window.location.reload();
+  }
+}
```
Comment (brainconnect93) [inline modules/Book.js:27]: - [ ] Great effort working on the awesome book project 👍 , but currently, in your project, I noticed I could not add or remove books, anytime a user tries to add a book using the form in the `add book` section it's not adding and in the `list` section it's not displaying any added book which makes it difficult to remove a book from the list. Kindly fix this

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
## STATUS: Changes Required :recycle:

### Hi @maov19, 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 
-  Has a proper Pull Request. ✔️  
- No Linter errors :heavy_check_mark: 
- Good use of Github flow ✔️ 

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by maov19** [comment]
Dear reviewer, 

The issue observed before was solved. You can see a screen recording below ⬇️ 
Please let me know if there is anything else I should work on.

awesome_es6

**Review by Munsa1 — APPROVED** [review]
Hi @maov19, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Passing Linters :heavy_check_mark: 
Correct GitHub Flow :heavy_check_mark: 
Descriptive ReadMe File :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**